arch/arm/boot/dts/overlays/README

```diff
@@ -5324,6 +5324,8 @@ Params: 2_8_inch                2.8" 480x640
         8_0_inch                8.0" 1280x800
         10_1_inch               10.1" 1280x800
         11_9_inch               11.9" 320x1480
+	13.3inch-30fps        13.3″ 1920x1080 2lane
+	13.3inch-60fps        13.3″ 1920x1080 4lane
```

Comment:
Replace the TABs with spaces, fix the indentation so it all lines up, and get the parameter names right!